### ISSUE TITLE
Allow menu.webApp to accept dynamic text

### ISSUE THREAD

**Issue by kiyasov** [grammyjs/menu#34]
# Current
 ```js
 webApp(text: string, url: string): this;
 ```

# Proposed 
```js
webApp(text: MaybeDynamicString<C>,  url: string): this;
webApp(text: string, url: string): this;
```

**Comment by KnorpelSenf** [comment]
Good idea, do you want to implement this?

**Comment by kiyasov** [comment]
> Good idea, do you want to implement this?

#35

**Comment by KnorpelSenf** [comment]
Released a while ago as 1.2.1.
